src/ServicePulse.Host/vue/src/views/MessageView.vue

```diff
@@ -0,0 +1,119 @@
+<script setup>
+import { ref, onMounted } from "vue";
+import { useRoute } from "vue-router";
+import { useFetchFromServiceControl } from "../composables/serviceServiceControlUrls";
+import NoData from "../components/NoData.vue";
+import TimeSince from "../components/TimeSince.vue";
+
+const route = useRoute();
+const id = route.params.id;
+const failedMessage = ref({});
+
+function loadFailedMessage() {
+  return useFetchFromServiceControl("errors/last/" + id)
+    .then((response) => {
+      if (response.status === 404) {
+        failedMessage.value = { notFound: true };
+      } else {
+        failedMessage.value = { error: true };
+      }
+      return response.json();
+    })
+    .then((data) => {
+      var message = data;
+      message.archived = message.status === "archived";
+      message.resolved = message.status === "resolved";
+      message.retried = message.status === "retryIssued";
+      failedMessage.value = message;
+    })
+    .catch((err) => {
+      console.log(err);
+      return;
+    });
+}
+
+function togglePanel(panelNum) {
+  if (failedMessage.value.notFound || failedMessage.value.error) return false;
+
+  failedMessage.value.panel = panelNum;
+  return false;
```

Comment:
```suggestion
  if (!failedMessage.value.notFound && !failedMessage.value.error) {
    failedMessage.value.panel = panelNum;
  }
  return false;
```

2 returns feels weird given that they're both return false.

Reply:
I accidentally made [this change](https://github.com/Particular/ServicePulse/pull/1384/commits/7f765854141863e510d6492c6e8d0f707da2c545#diff-eab870483f4a11e1169e4d2d0e5e24f4bee9d0220699a8550d7c8ee2438471c2R79-R81) in the other PR for the headers and the body that is branched off from this PR.